docs/content/doc/features/authentication.en-us.md

```diff
@@ -329,3 +329,22 @@ Before activating SSPI single sign-on authentication (SSO) you have to prepare y
   - You have added the URL of the web app to the `Local intranet zone`
   - The clocks of the server and client should not differ with more than 5 minutes (depends on group policy)
   - `Integrated Windows Authentication` should be enabled in Internet Explorer (under `Advanced settings`)
+
+## Reverse Proxy
+
+Gitea supports Reverse Proxy Header authentication, it will read headers as a trusted login user name or user email address. This hasn't been enabled default, you can enable it with
```

Comment:
```suggestion
Gitea supports Reverse Proxy Header authentication, it will read headers as a trusted login user name or user email address. This hasn't been enabled by default, you can enable it with
```